src/ansys/geometry/core/tools/repair_tools.py

```diff
@@ -644,11 +648,78 @@ def find_and_fix_simplify(
         parent_design = get_design_from_body(bodies[0])
         parent_design._update_design_inplace()
         message = RepairToolMessage(
-            response["success"],
-            response["created_bodies_monikers"],
-            response["modified_bodies_monikers"],
-            response["found"],
-            response["repaired"],
+            response.get("success"),
+            response.get("created_bodies_monikers"),
+            response.get("modified_bodies_monikers"),
+            response.get("found"),
+            response.get("repaired"),
+        )
+        return message
+
+    @min_backend_version(25, 2, 0)
```

Comment:
Will It be shipped with 25R2?

Reply:
After investigations done by @RyanJWard, it looks like the changes are in -- no need to modify the decorator! 🥳